api/src/test/java/org/apache/iceberg/transforms/TestBucketing.java

```diff
@@ -112,12 +112,59 @@ public void testSpecValues() {
         .as("Spec example: hash(2017-11-16T22:31:08) = -2047944441")
         .isEqualTo(-2047944441);
 
+    timestampVal = Literal.of("2017-11-16T22:31:08.000001").to(Types.TimestampType.withoutZone());
+    assertThat(BucketUtil.hash(timestampVal.value()))
+        .as("Spec example: hash(2017-11-16T22:31:08.000001) = -1207196810")
+        .isEqualTo(-1207196810);
+
     Literal<Long> timestamptzVal =
         Literal.of("2017-11-16T14:31:08-08:00").to(Types.TimestampType.withZone());
     assertThat(BucketUtil.hash(timestamptzVal.value()))
         .as("Spec example: hash(2017-11-16T14:31:08-08:00) = -2047944441")
         .isEqualTo(-2047944441);
 
+    timestamptzVal =
+        Literal.of("2017-11-16T14:31:08.000001-08:00").to(Types.TimestampType.withZone());
+    assertThat(BucketUtil.hash(timestamptzVal.value()))
+        .as("Spec example: hash(2017-11-16T14:31:08.000001-08:00) = -1207196810")
+        .isEqualTo(-1207196810);
+
+    Literal<Long> timestampNsVal =
+        Literal.of("2017-11-16T22:31:08").to(Types.TimestampNanoType.withoutZone());
+    assertThat(BucketUtil.hash(timestampNsVal.value()))
+        .as("Spec example: hash(2017-11-16T22:31:08) = -737750069")
```

Comment:
the spec examples don't seem to align with what's in the spec? Can you double check all of these please?

Reply:
Ahhhh these are from before commit 753aed569184890156f826347fbbf6eab843cf1b changed the bucketing. New tests were added in that commit and in commit eb5e38242348ebed6778432ead28182672bafc79, obviating all these. I've removed these now.

Reply:
do we have tests that refer to the examples in the spec?

Reply:
Yes, `testTimestampNanoPromotion added in commit 753aed5 and `testTimestampTzNanoPromotion` added in commit eb5e382.